docs/kernel.rst

```diff
@@ -11,7 +11,9 @@ Testing a new kernel
 
 The following steps should be performed for all of the `recommended hardware`_:
 
-#. Install the new kernel packages on your *Monitor Server*, then reboot. Verify with ``uname -r`` that you are using the new kernel.
+#. Install the new kernel packages on your *Monitor Server* using unattended-upgrades,
+   e.g. ``sudo unattended-upgrades --debug`` or wait for the automatic nightly upgrade.
```

Comment:
AFAIK `unattended-upgrades` alone does not update the package index, so should it be `sudo apt update` first?

Reply:
Yes, will add.